qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -95,11 +123,16 @@
             iso_label_chk_cmd = 'wmic logicaldisk get VolumeName |findstr -i ${pkg_name}'
             installer_chk_cmd = 'powershell -command "(Get-Item -path "%s:\\virtio-win-guest-tools.exe").VersionInfo.ProductVersion"'
         - driver_install:
+            only single_driver
         - driver_repair:
+            only all_drivers
             type = win_virtio_driver_installer_repair
         - driver_uninstall:
+            only all_drivers
+            driver_test_names = ''
```

Comment:
I am not sure about this part, @xiagao Please help to confirm, thanks.

Reply:
I think there is no need to add function test for the uninstall installer test.

Reply:
@xiagao Yes, `driver_test_names = ''` means to execute nothing for function test, in case other matrix will want to test something with `all_drivers`. If that is the case, here we want to keep it, but for sure only when other scripts really use `driver_test_names` and implement function test, not for now.